packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -431,7 +431,13 @@ export interface IFluidParentContext
 	 */
 	readonly scope: FluidObject;
 
+	/**
+	 * @deprecated this functionality has been removed.
```

Comment:
Nit: probably worth noting what consumers are intended to use instead (if applicable). Or explicitly call out that no replacement is intended.

Reply:
This is an internal functionality. Its unfortunately tagged as alpha / legacy. So, we don't really need to specify anything here.